pipelines/cdp/cdp_pipeline/options.py

```diff
@@ -0,0 +1,14 @@
+from argparse import ArgumentParser
+
+from apache_beam.options.pipeline_options import PipelineOptions
+
+
+class MyPipelineOptions(PipelineOptions):
+  @classmethod
+  def _add_argparse_args(cls, parser: ArgumentParser):
+    parser.add_argument("--transactions_topic", type=str)
+    parser.add_argument("--coupons_redemption_topic", type=str)
+    parser.add_argument("--project_id", type=str)
```

Comment:
This option may be redundant with the project option of Google Cloud Options, already included in Beam.